Libraries/LibWeb/SVG/SVGElement.cpp

```diff
@@ -28,6 +30,38 @@ void SVGElement::initialize(JS::Realm& realm)
     WEB_SET_PROTOTYPE_FOR_INTERFACE(SVGElement);
 }
 
+bool SVGElement::should_include_in_accessibility_tree() const
+{
+    bool has_title_or_desc;
```

Comment:
This should be initialized to false. I can't be 100% sure, since I don't have a linux machine to test on but that may be why gcc CI is complaining currently.